appIcons.js

```diff
@@ -187,6 +191,13 @@ const DockAbstractAppIcon = GObject.registerClass({
             }
         });
 
+        this.connect('notify::updating', () => {
+            const icon = this.icon._iconBin;
```

Comment:
For next time, I think it would have been nicer to use `this.bind_property_full`.

However, I feel this code should be better to be as part of `AppIconIndicators.IndicatorBase`, no?

Since the `appIcon` is only the logical representation of an icon, shouldn't touch its look.

Reply:
Now that I see the code with more detail, I don't think so: this code is in DockAbstractAppIcon, from which DockAppIcon and DockLocationAppIcon. Maybe it can be moved to DockAppIcon, but surely it has to be there, because it is what manages the appearance of the icon itself. IndicatorBase only manages the things that are over the icon. In fact, IndicatorBase manages the "updating" notification, and relays it to the corresponding DockAbstractAppIcon, which, I think, is the right way.